api/workAllocation2/util.ts

```diff
@@ -85,6 +92,28 @@ export function assignActionsToTasks(tasks: any[], view: any): any[] {
   return tasksWithActions;
 }
 
+/**
+ * NOTE: These comments are copied from the assignActionsToTasks method
+ * The below sets up actions on the cases, though it's expected this will change
+ * in the future - it should do fine for the MVP, though.
+ * @param tasks The tasks to set up the actions for.
+ * @param view This dictates which set of actions we should use.
+ */
+export function assignActionsToCases(cases: any[], view: any): any[] {
```

Comment:
There is a comment above a similar 'assignActionsToTasks' stating the following:
TODO: "Make this more cleverer" (AndyW)
 * The below sets up actions on the tasks, though it's expected this will change
 * in the future - it should do fine for the MVP, though.

On this basis, shall we keep this as a TODO for now or are there any changes you recommend I do now?